apis/v1alpha1/httproute_types.go

```diff
@@ -173,11 +197,14 @@ type HTTPRouteRule struct {
 	// Support: core
 	//
 	// +optional
-	Filters []HTTPRouteFilter `json:"filters"`
+	// +kubebuilder:validation:MaxItems=16
+	Filters []HTTPRouteFilter `json:"filters,omitempty"`
 
-	// Forward defines the upstream target(s) where the request should be sent.
+	// ForwardTo defines the backend(s) where matching requests should be sent.
 	// +optional
-	Forward *HTTPForwardingTarget `json:"forward"`
+	// +kubebuilder:validation:MinItems=0
```

Comment:
Should the same be applied to `MinItems=0` here since [this](https://github.com/kubernetes-sigs/service-apis/pull/336/files#r495361672) consensus was achieved?

Reply:
Removed, thanks!